crates/executor/src/estimate.rs

```diff
@@ -60,6 +72,9 @@ pub fn estimate(
                 fees.push(FeeEstimate {
                     gas_consumed: U256::from(tx_info.actual_fee.0) / gas_price.max(1.into()),
                     gas_price,
+                    data_gas_consumed: U256::from(tx_info.da_gas.l1_data_gas)
+                        / data_gas_price.max(1.into()),
```

Comment:
I don't think this is correct.
`tx_info.da_gas.l1_data_has` seems to be the actual gas consumption (at least based on how `GasVector::saturated_cost()` works).